<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEbsdReferenceFrameDialog</class>
 <widget class="QDialog" name="QEbsdReferenceFrameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Reference Frame Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="m_NoTransBtn">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>No/Unknown Transformation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_HEDMdefaultBtn">
       <property name="text">
        <string>HEDM - IceNine (Default)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="m_TSLdefaultBtn">
       <property name="text">
        <string>Edax - TSL (Default)</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_HKLdefaultBtn">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Oxford - HKL (Default)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Sample Transformation:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_SampleTransformationLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>false</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Euler Transformation:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_EulerTransformationLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>false</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>X Dim:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_XDim">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Y Dim:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_YDim">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>X Res:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_XRes">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Y Res:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_YRes">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 11px;
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/icons/images/Icons.qrc">
         <normaloff>:/information.png</normaloff>:/information.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QToolButton" name="zoomButton">
       <property name="maximumSize">
        <size>
         <width>16777</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Set a specific zoon level</string>
       </property>
       <property name="statusTip">
        <string>Selects a set magnification level</string>
       </property>
       <property name="text">
        <string>100 % </string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::UpArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomIn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Zoom in to the image</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/icons/images/Icons.qrc">
         <normaloff>:/zoom_in.png</normaloff>:/zoom_in.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Zoon out from the image</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/icons/images/Icons.qrc">
         <normaloff>:/zoom_out.png</normaloff>:/zoom_out.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fitToWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Fit the image to the display area</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/icons/images/Icons.qrc">
         <normaloff>:/fit_to_window.png</normaloff>:/fit_to_window.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="degToRads">
     <property name="text">
      <string>Convert Displayed Data to Radians 
(Imported Data will NOT be changed)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>IPF Reference Direction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="refDir">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Z</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QtSGraphicsView" name="m_EbsdView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtSGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtSGraphicsView.h</header>
   <slots>
    <slot>zoomIn()</slot>
    <slot>zoomOut()</slot>
    <slot>setZoomIndex(int)</slot>
    <slot>fitInView()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QEbsdReferenceFrameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QEbsdReferenceFrameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomIn</sender>
   <signal>clicked()</signal>
   <receiver>m_EbsdView</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>663</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomOut</sender>
   <signal>clicked()</signal>
   <receiver>m_EbsdView</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
